album/SECL-1907.toml

```diff
@@ -0,0 +1,22 @@
+[album]
+title = "Ripple Effect"
+artist = "春奈るな"
+date = 2016-05-25
+type = "normal"
+catalog = "SECL-1907"
```

Comment:
Your version should be [SECL-1908](https://www.sonymusic.co.jp/artist/Lunaharuna/discography/SECL-1908) if you're using animation cover.